frontend/mulighetsrommet-cms/deskStructures/previews/TiltakstypeOgTiltaksgjennomforingPreview.js

```diff
@@ -1,14 +1,29 @@
 import React, { useEffect, useState } from "react";
 import sanityClient from "part:@sanity/base/client";
+import Switch from "react-switch";
+import ReactColorSquare from "react-color-square";
 
 const client = sanityClient.withConfig({ apiVersion: "2021-10-21" });
 
-function MinHeight({ children }) {
-  return <div style={{ minHeight: "550px" }}>{children}</div>;
+function MarginBottom({ children }) {
+  return <div style={{ marginBottom: "4rem" }}>{children}</div>;
 }
 
+function Legend({ farge, children }) {
+  return (
+    <div style={{ display: "flex", alignItems: "center" }}>
+      <ReactColorSquare height={12} width={12} color={farge} />
```

Comment:
Du trenger ikke endre det nå, men bruk av slike bibloteker som `ReactColorSquare` er et fint eksempel på når man kan lage noe selv istedenfor å dra inn en ekstern avhengighet. Jo færre eksterne avhengigheter jo bedre, og i dette tilfelle så kan du lage en komponent som du feks styler med `display: inline-block` og setter en eksplisitt høyde og bredde på, samt en bakgrunnsfarge. Da har du i teorien firkantene med fargen du vil ha, uten at vi trenger å tenke på å oppdatere biblioteket i fremtiden. 

Hvis du vil kan jeg gjerne se på det sammen med deg hvis du er usikker.

Reply:
Endret det nå, takk for tips! 🤩